components/vivocalendar/sources/common/events.mjs

```diff
@@ -0,0 +1,8 @@
+export default {
+  APPOINTMENT_CREATED: "appointment_created",
+  APPOINTMENT_UPDATED: "appointment_updated",
+  APPOINTMENT_DESTROYED: "appointment_destroyed",
+  CUSTOMER_CREATED: "customer_created",
```

Comment:
The issue lists a `new-customer` webhook source. Could that be implemented with the "customer_created" event type?